packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-line-settings.tsx

```diff
@@ -69,6 +71,29 @@ const LockedLineSettings = (props: Props) => {
     // Check if the line has length 0.
     const isInvalid = kvector.equal(point1.coord, point2.coord);
 
+    function getPrepopulatedAriaLabel() {
+        let str = `${capitalizeKind} from (${point1.coord[0]}, ${point1.coord[1]}) to (${point2.coord[0]}, ${point2.coord[1]})`;
```

Comment:
I imagine this formatting style will be reused in the aria labels for many different locked figure types, so... what do you think about extracting a function like

```ts
function formatLockedPoint(point: LockedPointType): string {
    const [x, y] = point.coord;
    return `(${x}, ${y})`;
}
```

Reply:
I think the LockedPointType is only used in locked points and locked lines, so I personally don't think there's enough  usage of that yet to extract the coordinates out?